src/components/MenuContent/AboutUsSection.tsx

```diff
@@ -30,7 +30,7 @@ const AboutUsSection: React.FC<{
       ) : (
         <SectionHeader>About Us</SectionHeader>
       )}
-      <AboutCopy>{aboutUsCopy}</AboutCopy>
+      <ParagraphCopy>{aboutUsCopy}</ParagraphCopy>
```

Comment:
Renaming to `paragraphCopy` as this styled component is no longer used for just the "About us" part of the mega-menu/footer.